content/rdi/architecture.md

```diff
@@ -97,3 +127,28 @@ Debezium Server should run in warm High Availability topology in one of the foll
 - Running on 2 VMs in an active/passive setup. The failover orchestration can be done by tools such as [Pacemaker](https://clusterlabs.org/pacemaker/doc/).
 - Running as a Kubernetes pod where Kubernetes watches the pod liveliness and readiness and acts to recover nonfunctional pod
   It is important to note that Redis Data Integration does not keep any state on Debezium. All state is kept in Redis Data Integration.
+
+## Write behind functionality and architecture
+
+RDI write behind allows integration between Redis Enterprise (as the source of changes to data) to downstream databases or datastores.
```

Comment:
you wrote Write Behind before, be consistent throughout the documentation.

Reply:
it should be lower case all the time